packages/manager/modules/carrier-sip/src/index.js

```diff
@@ -0,0 +1,30 @@
+import angular from 'angular';
+
+// Peer dependencies.
+import '@ovh-ux/manager-core';
+import 'ovh-api-services';
+
+// Module dependencies.
+import cdr from './cdr';
+import dashboard from './dashboard';
+import endpoints from './endpoints';
+
+// Services.
+import service from './carrier-sip.service';
+
+// Styles.
+import 'ovh-ui-kit/dist/oui.css';
```

Comment:
This should be imported by host to avoid weird stylesheet reset if I'm correct (cc @jleveugle @cbourgois )

Reply:
It's done (moved to the application level instead of the module). Thanks!